src/js/redux/reducers/search/filters/locationFilterFunctions.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Created by michaelbray on 12/12/16.
+ */
+
+import { Set } from 'immutable';
+
+const updateSelectedLocations = (state, value, direction) => {
+    let updatedSet = new Set(state);
+
+    if (updatedSet.includes(value) && direction === 'remove') {
```

Comment:
Is `direction` necessary in either of these `if` blocks (and also in the action, etc.)? It seems to me that we can just check for membership in the set and if it exists, remove, otherwise add.